RNPComponents/RNPComponents.md

```diff
@@ -0,0 +1,47 @@
+# React Native Paper Component Templates
+
+This Document contains a set of components Templates around various [React Native Paper](https://callstack.github.io/react-native-paper/index.html) components.
+
+By default, the theming mechanism provided by RNP is very minimal and does not allow us to style components precisely the way we want for Brightlayer UI applications. To circumvent this issue, we create a template that you can use bring them in alignment with Brightlayer UI applications
+
+These components templates simplify this process so that the end user do not create the component designs to bring them in alignment with Brightlayer UI applications.
```

Comment:
These component templates simplify the process, so the Designer or Developer does not need to create the component designs to bring them in alignment with Brightlayer UI applications.

Reply:
just suggestion see what others say